rust/tests/simple_commit_test.rs

```diff
@@ -85,7 +83,8 @@ mod simple_commit_fs {
 
         let mut tx1 = table.create_transaction(None);
         tx1.add_actions(tx1_actions());
-        let result = tx1.commit_version(1, None).await.unwrap();
+        let commit = tx1.prepare_commit(None).await.unwrap();
+        let result = tx1.try_commit(&commit, 1).await.unwrap();
```

Comment:
That's how the low-level transaction API is going to be used instead of `commit_version`